Source/hydro/fourth_order.F90

```diff
@@ -532,96 +554,84 @@ subroutine states(idir, &
           ! the limiting loops are now over zones
           if (limit_fourth_order == 1) then
 
+             ! this is a loop over cell centers, affecting
+             ! j-1/2,R and j+1/2,L
```

Comment:
This comment needs to be fixed since this is the k loop.

Reply:
fixed